src/System.Console/tests/Color.cs

```diff
@@ -92,6 +92,68 @@ public static void NonRedirectedOutputDoesUseAnsiSequences()
         Assert.Equal(3, Encoding.UTF8.GetString(data.ToArray()).ToCharArray().Count(c => c == Esc));
     }
 
+    [Fact]
+    [PlatformSpecific(PlatformID.AnyUnix)]
+    public void VerifyInstalledTermInfosParse()
+    {
+        foreach (string location in ConsolePal.TermInfo.Database._terminfoLocations)
+        {
+            // Skip directories that dont exist
+            if (Directory.Exists(location) == false)
+                continue;
+
+            foreach (string term in Directory.EnumerateFiles(location, "*", SearchOption.AllDirectories))
+            {
+                if (term.ToUpper().Contains("README")) continue;
+                ConsolePal.TermInfo.Database db = ConsolePal.TermInfo.Database.ReadDatabase(Path.GetFileName(term));
+                Assert.True(db != null, string.Format("Could not parse terminfo file {0}", term)); // Don't use Assert.NotNull so we can print out the term info file on failure
```

Comment:
Does enumerating the files in a separate function and passing that in via a Theory give us a benefit over enumerating in the function itself? I only went with this direction because no other function needed to enumerate over the files, so it seemed contained. If you think it's best to use a Theory then I'm all for changing it :)